tests/licenses/licensesClient.test.ts

```diff
@@ -1198,14 +1199,14 @@ describe('LicensesClient', () => {
       .getLicensesClient()
       .setUrl(LICENSES_MOCK_URL);
 
-    it('should call getSchedulesTasks method (v1 without "schedulesTasks")', async () => {
+    it('should call getSchedulesTasks method (v1 with "schedulesTasks")', async () => {
       const licenseReference = 'XSP123456';
 
       nock(LICENSES_MOCK_URL)
         .get(`/licenses/${licenseReference}/scheduledTasks`)
         .reply(constants.HTTP_STATUS_OK, {
           status: 200,
-          data: PAYLOAD_SCHEDULES_TASKS,
+          data: PAYLOAD_GET_SCHEDULES_TASKS,
```

Comment:
```suggestion
          data: PAYLOAD_GET_SCHEDULED_TASKS,
```

Reply:
And you should probably remove the import

Reply:
C'est pour garder la compatibilité avec xsp-web qui attend "schedulesTasks"